features/admin.server-configurations.v1/api/governance-connectors.ts

```diff
@@ -108,6 +108,42 @@ export const useGovernanceConnectors = <
     };
 };
 
+/**
+ * Get details of a governance connector.
+ *
+ * @param categoryId - ID of the connector category
+ * @param connectorId - ID of the connector
+ * @returns the governance connector.
+ */
+export const useGovernanceConnectorDetails = <
```

Comment:
We follow the naming convention `useGet + resource name` for SWR hooks. Shall we follow the same here?

```suggestion
export const useGetGovernanceConnectorById = <
```